### ISSUE TITLE
Developper setup in contributing guidelines

### ISSUE THREAD

**Issue by zaccharieramzi** [google/jaxopt#319]
Hi,

I would like to contribute to the repo.
I could not find any instructions on how to setup my dev. env. like what linter is used and with which parameters (for example I see that you use 2 spaces identation like all google projects).

By any chance do you have that setup described somewhere? Happy to make a PR with it to include it in the contributing guidelines.

**Comment by mblondel** [comment]
We have some developer documentation [here](https://jaxopt.github.io/stable/developer.html) but we don't enforce a particular coding style yet. Feel free to submit PRs anyway :)

**Comment by zaccharieramzi** [comment]
Yes yes ofc! It's just that I have VSCode screaming at me with red underlines for the 2 spaces indentation and I was wondering if maybe there was a `.flake8` config file that I could use.
But yes submitted the PR anyway (#320)!

**Comment by zaccharieramzi** [comment]
Ah I only looked at lbfgs tests on my laptop sorry! Correcting right away.

I will squash the commits but there is an option on GitHub to do squash merge (and even an option at the repository level to force all merges to be squashed).

**Comment by mblondel** [comment]
We don't rely on github to merge but on a bot and we have to manually squash, sorry.
